src/ucp/wireup/wireup.c

```diff
@@ -292,6 +292,25 @@ ucp_wireup_find_remote_p2p_addr(ucp_ep_h ep, ucp_lane_index_t remote_lane,
     return UCS_ERR_UNREACHABLE;
 }
 
+ucp_lane_index_t
+ucp_wireup_ep_configs_use_same_lane(ucp_ep_config_key_t *ep_config_key,
+                                    ucp_ep_config_key_t *another_ep_config_key,
```

Comment:
- just `key1` and `key2` like `ucp_ep_config_lane_tl_is_equal`
 - and `ucp_wireup_ep_configs_use_same_tl_lane`
to be consistent

Reply:
fixed